internal/ui/view.go

```
@@ -591,12 +591,38 @@ func humanBytes(b uint64) string {
+	if len(runes) <= max {
 		return s
 	}
+	if max == 1 {
+		return "…"
+	}
 	return "…" + string(runes[len(runes)-max+1:])
 }
 
+// truncatePath keeps the tail of a path, cutting at a separator so a
+// tight file list shows `…/api/sessions.go` rather than `…ternal/api/…`.
+func truncatePath(path string, max int) string {
```

golangci-lint (2.12.2)

[error] param max has same name as predeclared identifier

(predeclared)